invenio_circulation/views.py

```diff
@@ -126,7 +127,6 @@ def __init__(self, serializers, ctx, *args, **kwargs):
         """Constructor."""
         super(LoanActionResource, self).__init__(
             serializers,
-            default_media_type=ctx.get('default_media_type'),
```

Comment:
`default_media_type` is not in `ctx`, thus this line is probably useless.

Reply:
Yes we could omit this line for now.